public/app/features/variables/pickers/OptionsPicker/actions.ts

```diff
@@ -72,7 +72,13 @@ export const filterOrSearchOptions = (searchQuery = ''): ThunkResult<void> => {
   };
 };
 
-export const commitChangesToVariable = (): ThunkResult<void> => {
+const setVariable = async (updated: any) => {
```

Comment:
This should be VariableWithMultiSupport, right?
```suggestion
const setVariable = async (updated: VariableWithMultiSupport) => {
```

Reply:
Great catch! Forgot to update the types.